main/include/platform.h

```diff
@@ -167,7 +167,7 @@ void platform_set_baud(uint32_t baud);
 
 #define PLATFORM_HAS_TRACESWO
 #define NUM_TRACE_PACKETS (128) /* This is an 8K buffer */
-#define TRACESWO_PROTOCOL 2     /* 1 = Manchester, 2 = NRZ / async */
+#define SWO_ENCODING 2     /* 1 = Manchester, 2 = NRZ / async */
```

Comment:
Note there's a "secret" third thing here now - 3, meaning "both". If you ever implement Manchester recovery into Farpatch, you'll be able to enable this to get both modes builds where the user can choose which mode is being used at runtime.

Reply:
Thanks for noticing that!